rule_comparison.md

```diff
@@ -1,109 +1,109 @@
+| required               |✅|✅|✅|✅|✅|
 
 ## Repeated
-| Constraint Rule | Go | GoGo | C++ | Java |
-| ---| :---: | :---: | :---: | :---: |
-| min\_items/max_items   |✅|✅|✅|✅|
-| unique                 |✅|✅|✅|✅|
-| items                  |✅|✅|❌|✅|
+| Constraint Rule | Go | GoGo | C++ | Java | Python | 
+| ---| :---: | :---: | :---: | :---: | :---: |
+| min\_items/max_items   |✅|✅|✅|✅|❌|
+| unique                 |✅|✅|✅|✅|❌|
+| items                  |✅|✅|❌|✅|❌|
 
 ## Maps
-| Constraint Rule | Go | GoGo | C++ | Java |
-| ---| :---: | :---: | :---: | :---: |
-| min\_pairs/max_pairs   |✅|✅|❌|✅|
-| no_sparse              |✅|✅|❌|❌|
-| keys                   |✅|✅|❌|✅|
-| values                 |✅|✅|❌|✅|
+| Constraint Rule | Go | GoGo | C++ | Java | Python | 
+| ---| :---: | :---: | :---: | :---: | :---: |
+| min\_pairs/max_pairs   |✅|✅|❌|✅|❌|
+| no_sparse              |✅|✅|❌|❌|❌|
+| keys                   |✅|✅|❌|✅|❌|
+| values                 |✅|✅|❌|✅|❌|
 
 ## OneOf
-| Constraint Rule | Go | GoGo | C++ | Java |
-| ---| :---: | :---: | :---: | :---: |
-| required               |✅|✅|✅|✅|
+| Constraint Rule | Go | GoGo | C++ | Java | Python | 
+| ---| :---: | :---: | :---: | :---: | :---: |
+| required               |✅|✅|✅|✅|❌|
 
 ## WKT Scalar Value Wrappers
-| Constraint Rule | Go | GoGo | C++ | Java |
-| ---| :---: | :---: | :---: | :---: |
-| wrapper validation     |✅|✅|✅|✅|
+| Constraint Rule | Go | GoGo | C++ | Java | Python | 
+| ---| :---: | :---: | :---: | :---: | :---: |
+| wrapper validation     |✅|✅|✅|✅|✅|
 
 ## WKT Any
-| Constraint Rule | Go | GoGo | C++ | Java |
-| ---| :---: | :---: | :---: | :---: |
-| required               |✅|✅|✅|✅|
-| in/not_in              |✅|✅|✅|✅|
+| Constraint Rule | Go | GoGo | C++ | Java | Python | 
+| ---| :---: | :---: | :---: | :---: | :---: |
+| required               |✅|✅|✅|✅|❌|
+| in/not_in              |✅|✅|✅|✅|❌|
 
 ## WKT Duration
-| Constraint Rule | Go | GoGo | C++ | Java |
-| ---| :---: | :---: | :---: | :---: |
-| required               |✅|✅|✅|✅|
-| const                  |✅|✅|✅|✅|
-| lt/lte/gt/gte          |✅|✅|✅|✅|
-| in/not_in              |✅|✅|✅|✅|
+| Constraint Rule | Go | GoGo | C++ | Java | Python | 
+| ---| :---: | :---: | :---: | :---: | :---: |
+| required               |✅|✅|✅|✅|✅|
+| const                  |✅|✅|✅|✅|✅|
+| lt/lte/gt/gte          |✅|✅|✅|✅|✅|
+| in/not_in              |✅|✅|✅|✅|✅|
 
 ## WKT Timestamp
-| Constraint Rule | Go | GoGo | C++ | Java |
-| ---| :---: | :---: | :---: | :---: |
-| required               |✅|✅|❌|✅|
-| const                  |✅|✅|❌|✅|
-| lt/lte/gt/gte          |✅|✅|❌|✅|
-| lt_now/gt_now          |✅|✅|❌|✅|
-| within                 |✅|✅|❌|✅|
+| Constraint Rule | Go | GoGo | C++ | Java | Python | 
+| ---| :---: | :---: | :---: | :---: | :---: |
+| required               |✅|✅|❌|✅|✅|
+| const                  |✅|✅|❌|✅|✅|
+| lt/lte/gt/gte          |✅|✅|❌|✅|✅|
+| lt_now/gt_now          |✅|✅|❌|✅|✅|
+| within                 |✅|✅|❌|✅|✅|
```

Comment:
Can you verify this chart is up-to-date? If not, please make sure you provide support for anything that already has complete support from the other languages (such as Any and Enums).

Reply:
Yup this is up-to-date 👍 
Added support for Enums. Will add support for others in next PR.